view/frontend/templates/js/replacejs.phtml

```diff
@@ -629,6 +629,13 @@ if (!$block->isEnabled()) return;
                     // send create order request
                     $.post(settings.create_order_url, params)
                         .done(function (data) {
+
+                            if (data.status !== 'success') {
+                                reject(new Error(data.message || 'Network request failed'));
+                                if (data.message) showBoltErrorMessage(data.message);
```

Comment:
@daisy1754 
Create order is called before button is clicked and iframe opened. It can return en error like "The provided amount is higher than required..."

Reply:
@daisy1754 
On the other hand other unwanted errors and exceptions can popup here. I am removing it.

Reply:
let's not show popup. When you reject promise we record errors; if specific errors happen too often we can add custom error message on Bolt side